config.json

```
@@ -1143,6 +1143,62 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
-  "tags": []
+  "key_features": [
+    {
+      "title": "Evolving",
+      "content": "Typescript is a modern and constantly evolving language supported by Microsoft."
+    },
+    {
+      "icon": "features-cross-platform",
+      "title": "Widely used",
+      "content": "Typescript is used by many frontend frameworks, and can be used in Javascript based ones as well."
+    },
+    {
+      "icon": "features-strongly-typed",
+      "title": "Strongly typed",
+      "content": "Typescript flexible typing system lets you describe what to expect of your code easily."
+    },
+    {
+      "icon": "features-multi-paradigm",
+      "title": "Multi-paradigm",
+      "content": "Mainly focused in OOP, it also has tools to support other paradigms like Functional Programming."
```

```suggestion
      "title": "Use any programming style",
      "content": "Use prototype-based, object-oriented, functional, or declarative programming styles, and more."
```

I think it might be worthwhile to just copy the JavaScript one. TypeScript doesn't really favour OOP of other styles, but has _less_ support for some patterns we use in JavaScript. That support _is_ coming.